{"sections":[],"abstract":[{"type":"text","text":"Error raised when the CRC reported by the remote device does not match."},{"text":" ","type":"text"},{"type":"text","text":"Service has done 3 attempts to send the data."}],"schemaVersion":{"major":0,"minor":3,"patch":0},"variants":[{"paths":["\/documentation\/nordicdfu\/dfuerror\/crcerror"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/NordicDFU\/documentation\/NordicDFU","doc:\/\/NordicDFU\/documentation\/NordicDFU\/DFUError"]]},"metadata":{"symbolKind":"case","roleHeading":"Case","modules":[{"name":"NordicDFU"}],"title":"DFUError.crcError","role":"symbol","externalID":"c:@M@NordicDFU@E@DFUError@DFUErrorCrcError","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"crcError"}]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/NordicDFU\/documentation\/NordicDFU\/DFUError\/crcError"},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"crcError","kind":"identifier"}]}]}],"references":{"doc://NordicDFU/documentation/NordicDFU":{"kind":"symbol","identifier":"doc:\/\/NordicDFU\/documentation\/NordicDFU","title":"NordicDFU","role":"collection","url":"\/documentation\/nordicdfu","abstract":[{"text":"A Swift library for updating firmware of nRF51 and nRF52 devices over Bluetooth LE.","type":"text"}],"type":"topic"},"doc://NordicDFU/documentation/NordicDFU/DFUError":{"kind":"symbol","abstract":[{"type":"text","text":"A DFU error enumeration."}],"type":"topic","url":"\/documentation\/nordicdfu\/dfuerror","navigatorTitle":[{"kind":"identifier","text":"DFUError"}],"fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"DFUError"}],"title":"DFUError","identifier":"doc:\/\/NordicDFU\/documentation\/NordicDFU\/DFUError","role":"symbol"},"doc://NordicDFU/documentation/NordicDFU/DFUError/crcError":{"abstract":[{"text":"Error raised when the CRC reported by the remote device does not match.","type":"text"},{"text":" ","type":"text"},{"type":"text","text":"Service has done 3 attempts to send the data."}],"type":"topic","url":"\/documentation\/nordicdfu\/dfuerror\/crcerror","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"crcError","kind":"identifier"}],"title":"DFUError.crcError","kind":"symbol","role":"symbol","identifier":"doc:\/\/NordicDFU\/documentation\/NordicDFU\/DFUError\/crcError"}}}